edx/edx-enterprise
# ENT-501: Add pagination for enterprise catalogs api endpoint

**Description:**  Enterprise Catalog List endpoint returns an array of catalogs, however the list is not paginated. This PR makes sure that Enterprise Catalog List endpoint is paginated.

**JIRA:** [ENT-501](https://openedx.atlassian.net/browse/ENT-501)

**Installation instructions:** Simply upgrade edx-enterprise.

**Testing instructions:**

1. Open enterprise catalogs page at `enterprise/api/v1/catalogs/`
2. Make sure that enterprise catalogs are paginated.

**Merge checklist:**

  **DO NOT** just add dependencies to `requirements/*.txt` files.

**Post merge:**
